src/applications/personalization/profile-2/components/TwoFactorAuthorizationStatus.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import recordEvent from 'platform/monitoring/record-event';
+import { mfa } from 'platform/user/authentication/utilities';
+
+import Verified from './Verified';
+
+const TwoFactorAuthorizationStatus = ({ isMultifactorEnabled, useSSOe }) => {
+  if (isMultifactorEnabled) {
+    return (
+      <Verified>
+        You’ve added an extra layer of security to your account with 2-factor
+        authentication.
+      </Verified>
+    );
+  }
+
+  const mfaHandler = useSSO => {
+    recordEvent({ event: 'multifactor-link-clicked' });
+    mfa(useSSO ? 'v1' : 'v0');
```

Comment:
Not part of this PR work, but if you find yourself checking `useSSO`, we should ask that team to create a layer of abstraction somehow because these floating strings seem frail.

Reply:
@ericbuckley @hinzed1127 might have thoughts about that? I believe one of those guys set up this system.

Reply:
@ncksllvn this is something I added a couple of weeks ago and part of what the VA Startup team is working on for the SSOe changes.  What do you mean by "floating strings"?

Reply:
I was referring to the `'v1'` and `'v0'` values. Totally just brainstorming, but it would be neat if these were abstracted somehow so that any logic functions apply it consistently. As in, maybe app state stores the login method so that `mfa()` applies it automatically, or just as constants somehow to add some meaning to `v0` vs. `v1`.